suse_migration_services/units/prepare.py

```diff
@@ -102,6 +102,35 @@ def main():
         [root_path, 'sys']
     )
     try:
+        # log network info as network-online.target is done at this point
+        log.info(
+            'All Network Interfaces {0}{1}'.format(
+                os.linesep, Command.run(
+                    ['ip', 'a'], raise_on_error=False
+                ).output
+            )
+        )
+        log.info(
+            'Routing Tables {0}{1}'.format(
+                os.linesep, Command.run(
+                    ['ip', 'r'], raise_on_error=False
+                ).output
+            )
+        )
+        log.info(
+            'DNS Resolver {0}{1}'.format(
+                os.linesep, Command.run(
+                    ['cat', '/etc/resolv.conf'], raise_on_error=False
+                ).output
+            )
+        )
+        log.info(
+            'Network Bonding {0}{1}'.format(
+                os.linesep, Command.run(
+                    ['cat', '/proc/net/bonding/bond*'], raise_on_error=False
+                ).output
+            )
+        )
```

Comment:
I don't think the bonding information are relevant. Also ```/proc/net/bonding``` only exists if bonding is active. So this would lead to an error in the log at any time no bond network is used. I would not log bonding data unless someone can tell me why this information is useful ?

Reply:
It seems relevant information for Martin.
I did test it before the PR and yes, no bonding info in my case.

Reply:
Given we provide the network information as proposed here, then the bonding information should be included if it exists.

Reply:
ok but this condition is missing:

```
cat /proc/net/bonding/bond*
cat: '/proc/net/bonding/bond*': No such file or directory
```

So the code should only run if /proc/net/bonding is present

Reply:
We have to make sure that i.e. 802.3ad bonding mode is active and bonding is setup correct. 
There are cases when you will not have a chance to communicate over the configured network in case the bonding mode is not setup properly. So for debugging "network does not work during live system" - this information would be very useful.. and based on the udev learning's maybe also the 70-persistent-net might be needed in the worst case....

Reply:
ok Martin, thanks for the update